<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>highlighter</class>
 <widget class="QDialog" name="highlighter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>256</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Decide on the transparency of 
the highlighter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="slider_alfa">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_pressure">
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_size">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="slider_size">
     <property name="styleSheet">
      <string notr="true">.QSlider {
    min-height: 40px;
    max-height: 40px;
}



.QSlider::handle:horizontal {
    background: gray;
    border: 5px solid gray;
    width: 25px;
    height: 50px;
    margin: -12px -24px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="same_data">
     <property name="text">
      <string>Same data for highlighter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_up">
     <property name="text">
      <string>Place the highlighter stroke under the text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
